src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started to work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped to work");

```

Comment:
The message 'Bulldozer stopped to work' should be more informative and indicate what type of machine has stopped working. Consider rephrasing to 'Bulldozer has stopped moving earth'.